src/pyscript/_generator.py

```diff
@@ -18,3 +21,38 @@ def file_to_html(input_path: Path, title: str, output_path: Optional[Path]) -> N
     output_path = output_path or input_path.with_suffix(".html")
     with input_path.open("r") as fp:
         string_to_html(fp.read(), title, output_path)
+
+
+def create_project(
+    app_name: str,
+    app_description: str,
+    author_name: str,
+    author_email: str,
+) -> None:
+    """
+    New files created:
+
+    manifest.toml - project metadata
+    index.html - a "Hello world" start page for the project.
+
+    TODO: more files to add to the core project start state.
+    """
+    context = {
+        "app_name": app_name,
+        "app_description": app_description,
+        "author_name": author_name,
+        "author_email": author_email,
+        "version": f"{datetime.date.today().year}.1.1",
+        "created_on": datetime.datetime.now(),
+        "id": str(uuid4()),
+    }
+    context["version"] = f"{datetime.date.today().year}.1.1"
```

Comment:
Just taking a peek and these three statements are already in the dict

Reply:
Ack... lemme fix that.